index.html

```diff
@@ -48,6 +48,12 @@ <h1 class="pl-heading pl-element-title">{{ post.title }}</h1>
                         <h2 class="pl-heading pl-tab-heading pl-element-preview-title">{{post.title}} Preview</h2>
 
                         <div class="pl-tab-content">
+                            {% if post.tags contains "icons" %}
```

Comment:
Nice addition, its great to have controls here. I may make a quick PR to abstract this out into preview controls UI